dev/com.ibm.websphere.appserver.features/visibility/protected/com.ibm.websphere.appserver.authFilter-1.0.feature

```diff
@@ -2,6 +2,7 @@
 symbolicName=com.ibm.websphere.appserver.authFilter-1.0
 WLP-DisableAllFeatures-OnConflict: false
 visibility=protected
--bundles=com.ibm.ws.security.authentication.filter
+-features=\
+   com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1,4.0,5.0"
```

Comment:
Should this depend on the private API feature instead of the public feature?  Same for the auto features then as well.

Reply:
It is quite possible this could depend on the private feature. I am not sure which it should though.

Reply:
It looks like @utle made a change with PR #13795 to change this feature already and has created a merge conflict with your PR.  He changed to use the API features like you did and switch to have auto features.   So looks like you need to clean up your PR to remove some changes to avoid the conflict and then rebase to pick up his changes and rebuild.

Reply:
I rebased with Ut's changes, and rebuilt. Everything looks good with these private features. I had some others failures related to not having any tests run in Lite mode, and am rebuilding with those changes.